src/templates/info-page.js

```diff
@@ -32,11 +32,16 @@ PageTemplate.propTypes = {
   contentComponent: PropTypes.func,
 };
 
-const Page = ({ data }) => {
+const Page = ({ data, location }) => {
   const { markdownRemark: post } = data;
 
   return (
-    <Layout>
+    <Layout
+      article
+      title={post.frontmatter.title}
+      pathname={location.pathname}
+      description={post.frontmatter.description || post.excerpt}
```

Comment:
I think this will always resolve to the post.excerpt since `frontmatter.description` isn't in the graphql query.

Reply:
Good catch! fixed now.